pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -49,6 +49,7 @@ type listUsersQuery struct {
 	wasDispatchThrottled       *atomic.Bool
 	wasDatastoreThrottled      *atomic.Bool
 	expandDirectDispatch       expandDirectDispatchHandler
+	datastoreThrottlingEnabled bool
```

Comment:
note: this one uses `Throttling` while the 2 below use `Throttle`, but that's how [batchcheck](https://github.com/openfga/openfga/blob/fb7b8e35ce5217b42dd9b487ba50ecc0358c8cfa/pkg/server/commands/batch_check_command.go#L34), [check](https://github.com/openfga/openfga/blob/fb7b8e35ce5217b42dd9b487ba50ecc0358c8cfa/pkg/server/commands/check_command.go#L39), and [listobjects](https://github.com/openfga/openfga/blob/fb7b8e35ce5217b42dd9b487ba50ecc0358c8cfa/pkg/server/commands/list_objects.go#L69) have it as well